toolium/behave/environment.py

```diff
@@ -34,6 +34,17 @@ def install_pytest_asserts():
 from toolium.behave.env_utils import DynamicEnvironment
 
 
+def _fail_first_step_precondition_exception(scenario):
```

Comment:
Why this method is not appended into the `DynamicEnvironment` class?

Reply:
Moved to DynamicEnvironment class.

Reply:
Remove unused method: _fail_first_step_precondition_exception

Reply:
fixed